frontend/src/components/Spinner/Spinner.tsx

```diff
@@ -5,13 +5,30 @@ import S from './styles';
 
 type Props = {
   text: string;
+  size?: string;
```

Comment:
```suggestion
  size?: 'small' | 'large';
```
If we do something like this, then the intellisense will be able to tell you the options.